diff --git a/includes/destinations.file.inc b/includes/destinations.file.inc
--- a/includes/destinations.file.inc
+++ b/includes/destinations.file.inc
@@ -193,7 +193,8 @@ class backup_migrate_destination_files extends backup_migrate_destination {
    */
   function check_web_dir($directory) {
     // If the directory is specified with an absolute path, strip the site root.
-    $directory = substr(drupal_realpath($directory), strlen(drupal_realpath($_SERVER['DOCUMENT_ROOT']) . base_path()));
+    //$directory = substr(drupal_realpath($directory), strlen(drupal_realpath($_SERVER['DOCUMENT_ROOT']) . base_path()));
+    $directory = substr(drupal_realpath($directory), strlen(DRUPAL_ROOT . '/'));
 
     file_create_htaccess($directory, TRUE);
 
